packages/yoga/src/Accordion/web/Accordion.jsx

```diff
@@ -133,6 +141,28 @@ const Accordion = ({
     max-height: ${({ isOpen }) => (isOpen ? '9999px' : '0')};
     overflow: hidden;
     transition: max-height 200ms ease-in-out;
+
+    ${({
+      theme: {
+        yoga: {
+          components: { accordion },
+        },
+      },
+    }) =>
+      small &&
+      css`
+        ${Content} {
+          padding-bottom: ${accordion.padding.small}px;
+        }
+      `}
+
+    ${hasHorizontalPadding === false &&
```

Comment:
What do you think about using like this? I think that's easier to read.
```suggestion
    ${!hasHorizontalPadding &&
```

Reply:
Fixed: https://github.com/gympass/yoga/pull/628/commits/87a7c80323654b91de836583300ad2d4a69ead13